src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -427,4 +435,124 @@ public boolean isReady() {
     public void setHealthChecker(HealthChecker healthChecker) {
         this.healthChecker = healthChecker;
     }
+
+    HttpOpenApiOperation SEND = new HttpOpenApiOperation(HttpOpenApiOperations.SEND, Level.INFO) {
```

Comment:
Why do you think that DEBUG is better? I would love to see at least information like method and path for each operation so send and send to partition as well.

Reply:
I'm just thinking what it's likely to look like under load, where we could expect send and fetch requests to dominate all the others, and in some respects those are the least interesting. So it would be nice to be able to easily filter those out. But maybe that's not the right way of looking at it. Maybe we should use a different logger (i.e. different logger category) for each operation. That would allow the user to relatively easily filter what's logged themselves using existing mechanisms. Wdyt?

Reply:
What I see is that we are creating a sort of matrix. 
Right now the user can set the logger level (INFO, DEBUG, ...) through the log4j file and it influences how each "class" of operation builds the log message. Actually, we are grouping operations in two "classes" which are the actual Java classes (`HttpOpenApiOperation` and `HttpOpenApiOperationTrace`).
At the same time, it's hardcoded (for now) the "logger category" for each command in order to log that message as info, debug and so on.
As far as I understood you want to make this logger category configurable as well by the user somehow?

So at the end we are saying for example:

Send command has a log message building behavior defined by `HttpOpenApiOperation` class but driven by what user set as loglevel. At same time, instead of LEVEL, DEBUG we can have CATEGORY_COMMAND_1, CATEGORY_COMMAND_2 and having for example Send falling into CATEGORY_COMMAND_2 and the user can say "print logs for commands in CATEGORY_XXX".
Doesn't it start to be confusing?

Reply:
What I meant was using something along the lines of `LoggerFactory.getLogger(operationId).info(...)` for these access log-style messages. That would make it easy for the _user_ to configure the logging level on a per-operation basis. That alleviates us from having to decide which operations should only log at debug, for example.

Reply:
After our discussion offline I made the change, but I think that at this point we need a few doc to describe how user can configure the logging level per-operation basis. I will do that in a different PR.

Reply:
So we would need to update even this file https://github.com/strimzi/strimzi-kafka-operator/blob/master/cluster-operator/src/main/resources/kafkaBridgeDefaultLoggingProperties adding something like:

```
log4j.logger.http.openapi.operation.healthy=WARN, out
log4j.additivity.http.openapi.operation.healthy=false
log4j.logger.http.openapi.operation.ready=WARN, out
log4j.additivity.http.openapi.operation.ready=false
```